static/scenes/railroad/js/game.js

```diff
@@ -34,22 +35,14 @@ class Game {
     this.renderer.shadowMap.enabled = true;
     container.appendChild(this.renderer.domElement);
 
+    await app.Scene.preload();
+    await app.Present.preload();
 
```

Comment:
Optional: just a nit that you can do Promise.all([app.Scene.preload(), app.Present.preload(), loadedScenePromise])

Thinking loadedScenePromise comes from:
const loadedScenePromise = gltfLoader.loadAsync('models/demo-scene-animated.glb');